drivers/sensor/iis2dh/iis2dh_spi.c

```diff
@@ -29,10 +29,9 @@ static struct spi_config iis2dh_spi_conf = {
 	.cs        = NULL,
```

Comment:
Remove iis2dh_spi_conf as its not used anymore... Also do we need to fix the `operations` to match what is define here in `IIS2DH_SPI`

Reply:
Removed, thanks!

Don't think I understand what you mean by fixing the `operations`?

Reply:
So in there the spi_config is set to:
`	.operation = (SPI_OP_MODE_MASTER | SPI_MODE_CPOL |
		      SPI_MODE_CPHA | SPI_WORD_SET(8)),
`

Seems like we should have the same settings passed to `SPI_DT_SPEC_INST_GET` in `IIS2DH_SPI`

Reply:
I see, thank you!